.github/workflows/cekit-action.yaml

```diff
@@ -0,0 +1,51 @@
+name: Publish cekit
+on:
+  workflow_dispatch
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Install CEKit
+        uses: cekit/actions-setup-cekit@v1.1.7
+
+      - name: Create folders
+        run: |
+          mkdir -p container-images/jdk17/runtime/
+          mkdir -p container-images/jdk17/builder/
+          mkdir -p container-images/jdk21/runtime/
+          mkdir -p container-images/jdk21/builder/
+
+      - name: Run CEKit dry-run for JDK17 runtime
+        run: cekit --descriptor ./runtime-image/image.yaml build --dry-run docker
+      - name: Copy files to container-images folder
+        run: |
+          cp -prf target/image/Dockerfile container-images/jdk17/runtime/
+
+      - name: Run CEKit dry-run for JDK17 builder
+        run: cekit --descriptor ./builder-image/image.yaml build --dry-run docker
+      - name: Copy files to container-images folder
+        run: 
+          cp -prf target/image/Dockerfile container-images/jdk17/builder   
+
+      - name: Run CEKit dry-run for JDK21 runtime
+        run: cekit --descriptor ./runtime-image/image.yaml build --overrides ./runtime-image/image-jdk21-overrides.yaml --dry-run docker
+      - name: Copy files to container-images folder
+        run: |
+          cp -prf target/image/Dockerfile container-images/jdk21/runtime        
+  
+      - name: Run CEKit dry-run for JDK21 builder
+        run: cekit --descriptor ./builder-image/image.yaml build --overrides ./builder-image/image-jdk21-overrides.yaml --dry-run docker
+      - name: Copy files to container-images folder 
+        run: |
+          cp -prf target/image/Dockerfile container-images/jdk21/builder
```

Comment:
At this point, we have an up-to-date Dockerfile in container-images/jdk21/builder but this file is referencing cekit modules (eg `COPY modules/jboss.container.user ...`. What'd be the build context to build this images? 
Does cekit dry-run also fetch cekit modules? Should they be added to the git commit below?